packages/components/src/index.ts

```diff
@@ -1,2 +1,57 @@
-// Example of exporting components to be consumed from main @kaizen/components
+// Components inside this package
 export * from "./KaizenProvider"
+
+// Re-exports which need to be moved into this package
+export * from "@kaizen/a11y"
+export * from "@kaizen/brand"
+export * from "@kaizen/brand-moment"
+export * from "@kaizen/button"
+export * from "@kaizen/component-base"
+export * from "@kaizen/date-picker"
+
+// Part of design tokens is already exported from KaizenProvider
+export {
+  makeCssVariableDefinitionsMap,
+  makeCSSVariableTheme,
```

Comment:
The inconsistency here is killing me!

Reply:
I can put it at the bottom, out of sight, out of mind lol